modules/bot/components/Controller.php

```diff
@@ -55,6 +55,11 @@ protected function getBotName()
         return $this->module->getBotName();
     }
 
+    protected function getBotApi()
+    {
+        return $this->module->botApi;
```

Comment:
```suggestion
        return $this->module->getBotApi();
```
Follow [this comment](https://github.com/opensourcewebsite-org/opensourcewebsite-org/pull/286/files#r388158693)